build.gradle.kts

```diff
@@ -32,6 +33,8 @@ dependencyLocking { lockAllConfigurations() }
 
 dependencies {
   implementation("org.springframework.boot:spring-boot-starter-data-mongodb-reactive")
+  implementation("org.springframework.boot:spring-boot-starter-actuator")
```

Comment:
why introduced in this PR?

Reply:
The actuator was needed for the readiness and liveness probes, next time I'll handle this kind of work in a separated PR